Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TriangularFEMForceField.inl

```diff
@@ -794,7 +794,7 @@ void TriangularFEMForceField<DataTypes>::computeStress(type::Vec<3, Real>& stres
         m_triangleUtils.computeDisplacementLarge(D, R_0_2, triangleInf[elementIndex].rotatedInitialElements, p[a], p[b], p[c]);
 
         // and compute postions of a, b, c in the co-rotational frame
-        Coord A = Coord(0, 0, 0);
+        Coord A = Coord(0, 0, 0); SOFA_UNUSED(A);
```

Comment:
What's the reason to keep `A`?

Reply:
It was just to keep consistency (it would be weird if there was a B and C without a A...)

Reply:
this is your Cartesian mind ;)